src/Microsoft.Azure.WebJobs.Host/Bindings/DefaultResolutionPolicy.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+using Microsoft.Azure.WebJobs.Host.Bindings.Path;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Resolution policy for { } in  binding templates. 
+    /// The default policy is just a direct substitution for the binding data. 
+    /// Derived policies can enforce formatting / escaping when they do injection. 
+    /// </summary>
+    internal class DefaultResolutionPolicy<TAttribute> : IResolutionPolicy<TAttribute> where TAttribute : Attribute
```

Comment:
><TAttribute>  [](start = 42, length = 13)

I'd remove this generic param. 99% of guys don't need it and will ignore it, and the 1% case that needs it can cast.